assets/js/src/form_editor/templates/selection.tsx

```diff
@@ -46,6 +46,11 @@ export default () => {
   const { selectTemplate } = useDispatch('mailpoet-form-editor-templates');
   return (
     <>
+      {categories.map((category) => (
+        templates[category.name].map((template) => (
+          <link key={`thumbnail_prefetch_${template.id}`} rel="preload" href={template.thumbnail} as="image" />
```

Comment:
I am not sure if this is the best way to preload the images. Chrome is yelling at us: https://d.pr/i/yj6kZX

Reply:
I'll reduce the preload so that we load only 4 thumbnails per category. That will reduce the number of requests (warnings) and the user should still see images instantly.